.pre-commit-config.yaml

```diff
@@ -67,7 +67,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/ansible/ansible-lint.git
-    rev: v4.1.1a0
+    rev: 7a2221a32968f12bee8494caf978ce150ad1494f
```

Comment:
Can you open an issue to remind us to change this back to a release (when one becomes available)?

Reply:
I'm going to backport this change up to skeleton-generic.  I'll modify the issue that is up there: https://github.com/cisagov/skeleton-generic/issues/19